UseCase47/kubernetes/kojofitness/deployment.yaml

```diff
@@ -0,0 +1,101 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: kojofitness
+  namespace: use-case-47
+spec:
+  template:
+    metadata:
+      labels:
+        app: kojofitness
+    spec:
+      containers:
+        - image: bearcode/kojofitness:UseCase47
+          name: kojofitness
+          resources:
+            limits:
+              memory: 50Mi
```

Comment:
When LS pods were configured to use only 64Mi of memory the pods couldn't even start up successfully. If this works, awesome, otherwise it might be too low

Reply:
`30Mi` was the minimum limit I found in my testing where the job would fail to fork.